OpenRA.Game/ChatLabels.cs

```diff
@@ -0,0 +1,26 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+namespace OpenRA
+{
+	public class ChatLabels : IGlobalModData
+	{
+		public readonly string BattlefieldControl = "Battlefield Control";
+		public readonly string Mission = "Mission";
+		public readonly string Debug = "Debug";
+		public readonly string All = "All";
```

Comment:
These prefixes are missing important context that might lead to them being (mis)used in other contexts, which might lead to them breaking when eventually translated.

Can we use `AllChat` (`ChatAll`?), `TeamChat`, etc?

Reply:
Yes, good point. The context comes from them being under the `ChatLabels` class but that context probably does not carry over to the translation framework (I should get familiar with that). So without that actually all of them are missing context and can be misused. 

What about a more generic `Labels` (`UILabels`?, `ChromeLabels`?) class and then giving each label context: `ChatMission`, `ChatDebug`, etc.

Reply:
Ah, I see. I missed the class name here.

Making this more generic (`UILogicStrings`?) would provide a framework for solving the second core point blocking #10475.

Reply:
I'm on board with getting rid of hard-coded strings and making them translatable. I will look around the code for other strings and give this a little more thought.

Reply:
:+1: to talking a look to get more context and tweaking this if appropriate to prepare, but IMO this PR should stay focused on the Lua API, with the scope creeps left to later PRs.